tests/validate_animations.test.js

```diff
@@ -7,7 +7,9 @@ const VALID_ANIMATIONS_DIR = './tests/animations/valid/';
 const INVALID_ANIMATIONS_DIR = './tests/animations/invalid/';
 const EXAMPLES_DIR = './docs/static/examples/';
 
-const ajv = new Ajv();
+const ajv = new Ajv({
+    strict: false
```

Comment:
I assume this is for "$version", you can add a custom keyword to keep it strict:

```js
keywords: [{keyword: "$version"}]
```